spec/features/api_spec.rb

```diff
@@ -1,9 +1,9 @@
 require "spec_helper"
 
-RSpec.describe "Suspend a new project with --api flag" do
+RSpec.describe "Suspend a new project with --api flag", type: :feature, autoclean: false do
   before(:all) do
     drop_dummy_database
-    remove_project_directory
```

Comment:
Instead of removing just the tmp project directory, the new overall approach is to clear the whole `tmp` directory. That is much more reliable and less error-prone. The consequence is that we will no longer run individual methods like `FakeGithub.clear!`, `FakeHeroku.clear!`, `remove_project_directory`, etc.